src/main/flight/gps_rescue.c

```diff
@@ -362,20 +362,25 @@ static void rescueAttainPosition()
     const int zVelocityDerivative = zVelocityError - previousZVelocityError;
     previousZVelocityError = zVelocityError;
 
-    const int16_t hoverAdjustment = (hoverThrottle - 1000) / ct;
-    altitudeAdjustment = constrain(altitudeAdjustment + (throttle.Kp * zVelocityError + throttle.Ki * zVelocityIntegral + throttle.Kd * zVelocityDerivative),
-                                    gpsRescueConfig()->throttleMin - 1000 - hoverAdjustment, gpsRescueConfig()->throttleMax - 1000 - hoverAdjustment);
+    const int16_t hoverAdjustment = (int16_t)((hoverThrottle - 1000) / ct);
```

Comment:
This retains previous logic but we could potentially change to:
```c
const float hoverAdjustment = (hoverThrottle - 1000) / ct);
```

Reply:
What about
```
const int16_t hoverAdjustment = lrintf((hoverThrottle - 1000) / ct);
```

Reply:
Yes rounding seems to make more sense. I think this boils down to whether to change `rescueThrottle` to float (which could make sense since `rcCommand` is already float), or we round as you suggest.

Reply:
Let's wait for @IllusionFpv as he wrote some parts of that code.

Reply:
yes, `hoverAdjustment`, `altitudeAdjustment` are supposed to be integers so the code worked with implicit truncation

Reply:
@IllusionFpv Okay we will keep them as integers, what do you think about rounding as @daleckystepan suggests?

Reply:
It is ok for me (and hoverAdjustment is used only as a starting throttle when rescue is triggered, then throttle will be computed by pid controller, btw we need to do some flight tests)

Reply:
Ok. Let's postpone this to BF 4.4. I will make another PR with two essential lines that give us more space on F411.